app/components/submission/ManuscriptMetadata/SubjectAreaDropdown.md

```diff
@@ -1,9 +1,11 @@
 ```js
-<ThemelessSubjectAreaDropdown
+const theme = require('@pubsweet/elife-theme').default
```

Comment:
Do we need default here?

Reply:
Yes

Because of `exports default myComponentName`, the theme prop looks like this
![image](https://user-images.githubusercontent.com/15656538/41360623-1cb376fe-6f25-11e8-80c9-1b786cdd3b7c.png) without `.default` on the end

Reply:
Wait, now I'm really confused. Why do you use `ThemelessSubjectAreaDropdown` just to pass in the theme component? Can't you use `SubjectAreaDropdown`? It will have the theme passed down from the Wrapper here https://github.com/elifesciences/elife-xpub/blob/90240af14c656f3fe4e29234615eab6bcc7bc7fe/styleguide/components/Wrapper.js

Also, why did you use a different name for the class without theme? And why did you export it? You can't use `ThemelessSubjectAreaDropdown` without theme in props, that would lead to bugs. I can't comment on the original PR but my suggestion here is to do this:

1. rename `ThemelessSubjectAreaDropdown` to `SubjectAreaDropdown` and don't export it, like so
```
class SubjectAreaDropdown extends React.Component {
...
```
2. export SubjectAreaDropdown, but with theme object added to context
```
export default withTheme(SubjectAreaDropdown)
```

Look at this https://github.com/elifesciences/elife-xpub/blob/develop/app/components/ui/atoms/icons/Upload.js

Then the README should be changed to reflect that.

Reply:
See the conversation with @g-sam below

Reply:
yeah, mihail is right -- I forgot it was already wrapped